ktlint-bom/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+plugins {
+    `java-platform`
+    id("ktlint-publication")
+}
+
+publishing.publications.named<MavenPublication>("maven") {
+    from(components["javaPlatform"])
+}
+
+val excludeList =
+    listOf(
+        "ktlint-bom",
+        "ktlint-ruleset-template",
+        "ktlint-test-ruleset-provider-v2-deprecated",
+    )
```

Comment:
I think that the klint-api-consumer should be added as well. It is just an example of how to create an API-consumer. It is not meant to be used as parent dependency.

Also, the ktlint-ruleset-test-tooling is not something that typically should be included in an API-consumer project.

Reply:
I saw you added `POM_ARTIFACT_ID`s for those 2 modules, if so I'll exclude them and remove `POM_ARTIFACT_ID`s of them.

Reply:
5751031a2b019f6459e5bdcc7f811ae6ad616370
34e4d92f6e55c2d6274f81e43262f979f2910538